### DISCUSSION TITLE
Custom userChrome.css when using flatpak version

### DISCUSSION THREAD

**Discussion by Boris00035** [Floorp-Projects/Floorp#1239] [Feature Request]
Title speaks for itself, trying to use a custom stylesheet when using the flatpak. Saw some people online that made it work with the firefox flatpak, but could not extrapolate a way that worked for floorp. 

Was doubtful to ask it here or create an issue. Is it possible to use a custom userChrome.css with the flatpak version?
Thanks for any help and maintaining this!

**Comment by NeoSahadeo** [comment]
Configs will usually exist at `~/.var/app/one.ablaze.floorp/<profile-name>/chrome` for flatpak. The default location in the about:profiles doesn't work.
